soql-server-pg/src/main/scala/com/socrata/pg/server/QueryServer.scala

```diff
@@ -381,11 +381,20 @@ class QueryServer(val dsInfo: DSInfo, val caseSensitivity: CaseSensitivity, val
     }
   }
 
+  private def writeRollupHeader(header: String, ru: Option[RollupName], rus: Seq[RollupInfo])(resp: HttpServletResponse): Unit = {
+    val combined = ru.toSeq.map(_.underlying) ++
+      rus.map(ru => s"${(new TableName(ru.copyInfo.datasetInfo.resourceName.get)).nameWithoutPrefix}.${ru.name.underlying}")
```

Comment:
can resourceName be empty?

Reply:
resourceName was added after the initial nbe architecture.  For ease of transition, it wasn't made required in the database scheme.  But it is essentially required.